### PR TITLE
Mqtt connection, object avoidance pending

### PR DESCRIPTION
**Pull request by darmacuong**
Setup for manual control and mqtt connection, pending for object avoidance rework

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

**Review by LIYUHAN22 — APPROVED** [review]
this add the lib for MQTT.h

File: AndroidApp/app/src/main/java/com/example/androidapp/MainActivity.java [hunk 1]
```diff
@@ -89,16 +233,103 @@ public boolean onTouch(View v, MotionEvent event) {
...
+
+                }
+            }
+
+            @Override
+            public void deliveryComplete(IMqttDeliveryToken token) {
+
+            }
+        });
+        */
```
Comment (platisd) [inline AndroidApp/app/src/main/java/com/example/androidapp/MainActivity.java:332]: Why are you adding commented out code? (which is also outdated and wrong)

File: arduino/smartcar/smartcar.ino [hunk 2]
```diff
@@ -1,9 +1,326 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (platisd) [inline arduino/smartcar/smartcar.ino:326]: :scream:

File: sprint1/sprint1.ino [hunk 3]
```diff
@@ -1,5 +1,14 @@
 #include <Smartcar.h>
 #include <HardwareSerial.h>
+#include <vector>
```
Comment (platisd) [inline sprint1/sprint1.ino:3]: Why is this sketch here? Is it used? If not then remove it. If yes, then you **must** build it on CI.

**Pull request merged**